code/game/machinery/atmos_field_gen.dm

```diff
@@ -0,0 +1,289 @@
+#define ACTIVE_WANTPOWER 1
+#define ACTIVE 2
```

Comment:
these define names need to be more specific, they WILL bleed with others in the code-base someday. also, you should define the default case